package.json

```diff
@@ -143,7 +144,7 @@
         "./Pagination": "./dist/Pagination.js",
         "./Notice": "./dist/Notice.js",
         "./Modal": "./dist/Modal.js",
-        "./MainNavigation": "./dist/MainNavigation/index.js",
+        "./MainNavigation": "./dist/MainNavigation\\index.js",
```

Comment:
```suggestion
        "./MainNavigation": "./dist/MainNavigation/index.js",
```

Reply:
Oops I'm sorry I didn't see that, I guess you're not supposed to contribute from a Windows machine with prettier 😬